res/controllers/Keyboard-Default-without-test.kbd.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please add this new file to windows NSIS uninstall. see http://mixxx.org/wiki/doku.php/contributing_mappings#windows_installer_update